po/de.po

```diff
@@ -2127,9 +2117,8 @@ msgid "physically delete"
 msgstr "physisch von der Festplatte löschen"
 
 #: ../src/control/jobs/control_jobs.c:750
-#, fuzzy
 msgid "physically delete all files"
-msgstr "physisch von der Festplatte löschen"
+msgstr "alle Dateien physisch von der Festplatte löschen"
```

Comment:
Ich würde physisch weglassen,

Reply:
Ich hätte hier weder im Englischen Original _physically_ noch in der Deutschen Übersetzung _physisch_ geschrieben. Ich wollte meine Änderung hier aber auf ein Minimum beschränken, weshalb ich das _physisch_ habe stehen lassen, kann von mir aus gern weg.